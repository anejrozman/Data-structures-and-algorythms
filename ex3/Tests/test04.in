12,17,5
9.875222481320135,5.608272891264155,4.543198790991238,6.207786693463847,5.8785847282515045,5.037012899366488,3.1121161349795,6.768586955848616,3.5285319257977523,6.831594802649595,1.3759972172315398,9.14138473517359
9,8,385.952
6,5,129.257
3,4,785.706
4,10,532.826
6,4,538.063
4,5,300.295
1,4,183.213
4,7,696.172
3,8,698.335
5,7,571.431
10,11,188.783
7,8,580.075
2,7,478.562
4,0,617.817
9,10,491.491
1,10,621.843
10,2,885.67
