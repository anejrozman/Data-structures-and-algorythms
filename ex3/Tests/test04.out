4,145.6454
4,142.2524
2,30.751
1,0
1,0
